test/health/server-health.test.js

```diff
@@ -0,0 +1,51 @@
+/* eslint-disable no-console */
+
+// this basic server health check is meant to be run before integration tests
+// it should be run with the same environment variables as the integration tests
+// and operate in the same way as the integration tests
+
+const SeleniumHelper = require('../integration/selenium-helpers.js');
+
+const rootUrl = process.env.ROOT_URL || (() => {
+    const ROOT_URL_DEFAULT = 'https://scratch.ly';
+    console.warn(`ROOT_URL not set, defaulting to ${ROOT_URL_DEFAULT}`);
+    return ROOT_URL_DEFAULT;
+})();
+
+jest.setTimeout(60000);
+
+describe('www server health check', () => {
```

Comment:
I added this test one day when staging wasn't feeling healthy. I figured if we know the integration tests will all fail, there's no point in even starting them. Some day, we'll be able to run integration tests without deploying... when that day comes, this file can be deleted.